src/pe_mailer/stats_message.py

```diff
@@ -0,0 +1,109 @@
+"""This module contains the StatsMessage class."""
+
+# Standard Python Libraries
+import datetime
+
+# Third-Party Libraries
+import pystache
+
+from .message import Message
+
+
+class StatsMessage(Message):
+    """An email message containing summary statistics for a run.
+
+    Static attributes
+    -----------------
+    Subject : str
+        The mustache template to use when constructing the message
+        subject.
+
+    TextBody : str
+        The mustache template to use when constructing the plain text
+        message body.
+
+    HtmlBody : str
+        The mustache template to use when constructing the HTML
+        message body.
+
+    """
+
+    Subject = "cyhy-mailer summary from {{date}}"
+
+    TextBody = """Greetings!
+
+Here is the cyhy-mailer summary from the run ending at {{date}}:
+{{#strings}}
+* {{string}}
+{{/strings}}
+
+Please direct feedback and questions to ncats-dev@beta.dhs.gov and/or the cyhy-mailer GitHub project.
+
+Regards,
+The VM Development Team
+Cybersecurity and Infrastructure Security Agency (CISA)
+ncats-dev@beta.dhs.gov
+"""
+
+    HtmlBody = """<html>
+<head></head>
+<body>
+<div style="font-size:14.5">
+<p>Greetings!</p>
+
+<p>
+Here is the cyhy-mailer summary from {{date}}:
+<ul>
+{{#strings}}
+<li>{{string}}</li>
+{{/strings}}
+</ul>
+</p>
+
+<p> Please direct feedback and questions to <a
+href="mailto:ncats-dev@beta.dhs.gov">the VM Development Team</a>
+and/or the <a
+href="https://github.com/cisagov/cyhy-mailer">cyhy-mailer GitHub
+project</a>.</p>
+
+<p>
+Regards,<br>
+The VM Development Team<br><br>
+Cybersecurity and Infrastructure Security Agency<br>
+<a href="mailto:ncats-dev@beta.dhs.gov">ncats-dev@beta.dhs.gov</a>
```

Comment:
Same as above:
```suggestion
<p> Please direct feedback and questions to <a
href="mailto:???@???">the PE Development Team</a>
and/or the <a
href="https://github.com/cisagov/pe-reports">pe-reports GitHub
project</a>.</p>

<p>
Regards,<br>
The P&E Development Team<br><br>
Cybersecurity and Infrastructure Security Agency<br>
<a href="mailto:???@???">???@???</a>
```

Reply:
Bad copy over - updated in commit: 6f8b5d8